x-pack/plugins/enterprise_search/public/applications/shared/layout/beta.scss

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+ @include euiBreakpoint('m', 'l', 'xl') {
+  .kbnPageTemplateSolutionNav {
```

Comment:
Note: we currently can't pass a custom className into Kibana's solution/side nav. It overrides the existing CSS and breaks styling/UX. I reported the issue to Caroline, but she's on PTO right now and I don't think we should wait for that to get fixed vs just hooking into the existing CSS class. Plus this is eventually going anyway within ~2 minors, so it doesn't need to be super beautiful